querydsl-sql-spatial/src/main/java/com/querydsl/sql/spatial/SQLServerGeometryType.java

```diff
@@ -72,7 +74,7 @@ public String getLiteral(Geometry geometry) {
         if (geometry.getSRID() > -1) {
             return "geometry::STGeomFromText('" + str + "', " + geometry.getSRID() + ")";
         } else {
-            return "geometry::STGeomFromText('" + str + "')";
+            return "geometry::STGeomFromText('" + str + "', 4326)";
```

Comment:
I would probably name that magic constant, but I think the rest of the PR looks good.